Topic 3/src/test/java/bootcamp/exercises/DecimalTest.java

```diff
@@ -0,0 +1,29 @@
+package bootcamp.exercises;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class DecimalTest {
+
+	Decimal decimal= new Decimal();
+	
+	@Test
+	public void test1(){
+		assertEquals(decimal.toRoman(6),"VI");
+	}
+	
+	@Test
+	public void test2(){
+		assertEquals(decimal.toRoman(1),"I");
+	}
+	
+	/**
+	 * If the entered number is invalid, shoot me an exception.
+	 */
+	@Test(expected= OverflowException.class)
+	public void test3(){
+		decimal.toRoman(0);
+	}
+
```

Comment:
[High] You should add more test cases here, to be sure your algorithm it's working as expected. Think about or possible scenarios that will help you to be sure your class works ok.